src/main/java/com/chariotsolutions/meetupnotifier/AppBeanImpl.java

```diff
@@ -0,0 +1,51 @@
+package com.chariotsolutions.meetupnotifier;
+
+
+import com.google.api.services.calendar.model.Event;
+
+import com.chariotsolutions.meetupnotifier.meetup.api.GoogleConverter;
+import com.chariotsolutions.meetupnotifier.meetup.api.MeetupQuery;
+import com.chariotsolutions.meetupnotifier.meetup.api.MeetupQueryExecutor;
+
+import com.chariotsolutions.meetupnotifier.meetup.api.messages.Result;
+import com.chariotsolutions.meetupnotifier.meetup.api.messages.Results;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+@EnableScheduling
+public class AppBeanImpl {
+
+  private static final Logger logger = LoggerFactory.getLogger(AppBeanImpl.class);
+
+  private String[] groups;
+
+  @Scheduled(fixedRate = 60000L, initialDelay = 4000L)
```

Comment:
Well in "production" you wouldn't have DEBUG level logging enabled anyway  - that's typically reserved for when a developer is tracking down or trying to reproduce a problem.  They can turn on DEBUG (or TRACE) level logging to see more information.  So when you write debug log statements, think of the audience as software developers, not users or administrators.